CHANGE_LOG

```diff
@@ -1,3 +1,47 @@
+v0.40.0rc1 (March 30, 2023)
+---------------------------
+
+This release predominantly upgrades to LLVM 14 and Python 3.11.
+The minimum supported python version is now Python 3.8.
+
+Note: A bug was discovered in LLVM's RuntimeDyldELF on the Aarch64 platform that
+can cause segfaults when cross module symbols are linked. It is necessary
+for JIT users to build LLVM with the patch added in 
+`PR#926 <https://github.com/numba/llvmlite/pull/926>`_.
+
+Pull-Requests:
+
+* PR `#827 <https://github.com/numba/llvmlite/pull/827>`_: Add more LLVM pass bindings (`apmasell <https://github.com/apmasell>`_)
+* PR `#830 <https://github.com/numba/llvmlite/pull/830>`_: Add LLVM 14 support (`apmasell <https://github.com/apmasell>`_)
+* PR `#860 <https://github.com/numba/llvmlite/pull/860>`_: the git tag for the RC needs an rc1 suffix (`esc <https://github.com/esc>`_)
+* PR `#869 <https://github.com/numba/llvmlite/pull/869>`_: bump max Python version to 3.11 (`esc <https://github.com/esc>`_ `sklam <https://github.com/sklam>`_)
+
+* PR `#876 <https://github.com/numba/llvmlite/pull/876>`_: Remove `llvmlite.llvmpy` after deprecation (`apmasell <https://github.com/apmasell>`_)
```

Comment:
```suggestion
* PR `#876 <https://github.com/numba/llvmlite/pull/876>`_: Remove ``llvmlite.llvmpy`` after deprecation (`apmasell <https://github.com/apmasell>`_)
```

Reply:
fixed in https://github.com/numba/llvmlite/pull/930/commits/e8362019e9a288e0265035f80dda4026214b24e7